containers/Ideas/ExploreListItem.js

```diff
@@ -16,7 +16,10 @@ export const ExploreListItem = (props) => {
   const clockIcon = (<IconSLI name='clock' color={COLORS.grey.medium} />)
 
   return (
-    <TouchableOpacity accessibilityRole="button" onPress={() => props.action(props.item)}>
+    <TouchableOpacity
+      accessibilityRole="button"
+      onPress={() => props.action(props.item)}
+      delayPressIn={100}>
```

Comment:
I just wanted to see if this would work o proper device, if so one less thing to do but we should maybe look at a refactor using `Pressable`when we look at the story